### ISSUE TITLE
Can't download extensions

### ISSUE THREAD

**Issue by metroid112** [inorichi/tachiyomi#1848]
Version r1463 Android 9 Xiaomi Mi A2

![](https://i.imgur.com/dmlcp2i.png)

A few days ago I stopped to be able to download extensions through the app. I can download the APK and install it but any download through the app is eternally on "Pending"

**Comment by as280093** [comment]
check without vpn

**Comment by inorichi** [comment]
We use Android's download manager for extensions, same as the play store. Check if you're able to download apps from the store too.

**Comment by inorichi** [comment]
Check this too https://stackoverflow.com/a/53026503/3263582

**Comment by metroid112** [comment]
I can download apps from Google Play Store just fine. I disabled the 1.1.1.1 VPN app and tried but it was still the same.

Then I actually closed Tachi to restart after disabling and they would download just fine. So it's actually the 1.1.1.1 app having issues with this.

The weirdest thing is that my wi-fi at home has been using the 1.1.1.1 DNS directly from router settings for like a year now and I could download without problems. It seems using the VPN from the app is conflicting the downloads.
